### PR TITLE
Add pavan kumar gopidesu into committer list

### PR DESCRIPTION
**Pull request by gopidesupavan**
---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/project.rst [hunk 1]
```diff
@@ -84,6 +84,7 @@ Committers
 - Phani Kumar (@phanikumv)
 - Pierre Jeambrun (@pierrejeambrun)
 - Ping Zhang (@pingzh)
+- Pavan Kumar Gopidesu (@gopidesupavan)
```
Comment (kaxil) [inline docs/apache-airflow/project.rst]: Could you move your name after `Patrick` so it stays alphabetically sorted

Comment (kaxil) [inline docs/apache-airflow/project.rst]:
```suggestion
- Pavan Kumar Gopidesu (@gopidesupavan)
- Phani Kumar (@phanikumv)
- Pierre Jeambrun (@pierrejeambrun)
- Ping Zhang (@pingzh)
```

**Comment by kaxil** [comment]
@gopidesupavan Could you add your name and details on airflow-site too please: https://github.com/apache/airflow-site/blob/main/landing-pages/site/data/committers.json

**Review by kaxil — APPROVED** [review]

**Review by amoghrajesh — APPROVED** [review]

**Comment by gopidesupavan** [comment]
> @gopidesupavan Could you add your name and details on airflow-site too please: https://github.com/apache/airflow-site/blob/main/landing-pages/site/data/committers.json

Thank you Kaxil for making changes 😄 . sure will update in airflow-site

**Pull request merged**